libsolidity/interface/SMTSolverCommand.cpp

```diff
@@ -37,6 +29,8 @@ void SMTSolverCommand::setEldarica(std::optional<unsigned int> timeoutInMillisec
 {
 	m_arguments.clear();
 	m_solverCmd = "eld";
+	m_arguments.emplace_back("-hsmt");
+	m_arguments.emplace_back("-in");
```

Comment:
Not to be too pedantic, but leaving a comment as to what these flags do would be nice :)